src/bin/commands/demux.rs

```diff
@@ -93,6 +93,7 @@ impl ReadSet {
     const SPACE: u8 = b' ';
     const COLON: u8 = b':';
     const PLUS: u8 = b'+';
+    const READ_NUMBERS: &[u8] = b"12345678";
```

Comment:
minor (non-blocking) suggestion:

Start at `0` and add `9`.

Then, below, you can just write:

```
writer.write_all(&[Self::READ_NUMBERS[read_num]])?;
```



```suggestion
    const READ_NUMBERS: &[u8] = b"0123456789";
```